README.md

```diff
@@ -35,9 +155,104 @@ make dev
 
 You can now access the entry point at <http://localhost:8080>, or view the console at <http://localhost:8081>.
```

Comment:
I can see how `8080` relates to https://github.com/libero/article-store/pull/261/files#diff-04c6e90faac2675aa89e2176d2eec7d8R92, but is there a way of configuring a different port from `8081` if desired?

Reply:
These actually refer to the local Docker Compose config: https://github.com/libero/article-store/blob/1bdf8a2a1069534ee0cb3fef96053c74e6debe38/.docker/docker-compose.yml#L14 and https://github.com/libero/article-store/blob/1bdf8a2a1069534ee0cb3fef96053c74e6debe38/.docker/docker-compose.yml#L21

Not customisable at the moment, but can be.

Reply:
Might be worth making that more explicit? Whenever I see port numbers in a local dev context I always want to know if / how I can change them.

Reply:
Let's follow-up with a change to make them customisable.

Reply:
Opened https://github.com/libero/publisher/issues/383.